webextensions/api/windows.json

```diff
@@ -855,7 +855,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": "≤61"
+                    "version_added": "10"
```

Comment:
✅ https://github.com/chromium/chromium/commit/cdc6a8eed6416ab6efb3195186f4a26696ea2af3 in [10.0.601.0](https://github.com/chromium/chromium/releases/tag/10.0.601.0) before 10.0.648.